### PR TITLE
Complete Travis CI test system 

### PR DESCRIPTION
**Pull request by sinban04**
I made a finished Travis CI test system for ANT.
I made it as a script like Samsung's iotjs.

### CODE REVIEW

File: tools/install-deps-rpi.sh [hunk 1]
```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
```
Comment (RedCarrottt) [inline tools/install-deps-rpi.sh]: As I know, it is duplicated with ```target/raspberry-pi2_3/install-deps-raspberry-pi2_3.sh```. I suggest that you remove ```target/raspberry-pi2_3/install-deps-raspberry-pi2_3```.

File: tools/install-deps-rpi.sh [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+# Step 1. Install packages by apt-get
+print_progress 1 "Install dependent packages..."
+sudo apt-get update
+sudo apt-get -y install make g++-4.8                                          \
+  wiringpi libdbus-1-dev glib-2.0 libdbus-glib-1-2 bison byacc                 \
+  libdbus-glib-1-2-dbg libdbus-glib-1-dev zip sqlite3 libsqlite3-dev cmake    \
+  libgtk2.0-dev pkg-config libavcodec-dev libavformat-dev libswscale-dev git  \
+  python-dev python-numpy libjpeg-dev libpng-dev libtiff-dev libjasper-dev    \
+  libdc1394-22-dev automake libtool libssl-dev libnl-3-dev libnl-genl-3-dev   \
+  python3 udhcpd libv4l-dev libboost-dev libboost-serialization-dev gstreamer1.0
```
Comment (LeeHayun) [inline tools/install-deps-rpi.sh]: Add ```libopencv-dev```

File: tools/install-deps-rpi.sh [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+# Step 5. Build and install libxml2-2.9.4-rc2
+print_progress 5 "Build and install libxml2-2.9.4-rc2..."
+cd ${ANT_REPO_DIR}/dep/libxml2-2.9.4-rc2
+./autogen.sh
+./configure --prefix=/usr/local/xml
+make
+sudo make install
+
+# Step 6. Build and install opencv-3.0.0
```
Comment (LeeHayun) [inline tools/install-deps-rpi.sh]: Delete building and installation OpenCV step.

File: tools/install-deps-rpi.sh (hunk 1 above)
Comment (sinban04) [reply tools/install-deps-rpi.sh]: I leave it for the compatibility with the previous version. I just added the test system and I think the sub following should be done step by step.

File: tools/install-deps-rpi.sh (hunk 2 above)
Comment (sinban04) [reply tools/install-deps-rpi.sh]: I think this should be done in another commit, as you pulled request in #26 .
My PR is just about the test system. please do the modification in your PR.

**Comment by sinban04** [comment]
About the License, i added the license information of Samsung/iotjs inside the files i refer.
I think the License file in the project root directory should contain the license information about the entire our project. 
Some minor licenses related to some files should be applied to those source files.
There are many dependent libraries, but we don't include every license information in the License file.

**Comment by RedCarrottt** [comment]
@sinban04 
In the ```LICENSE``` file of IoT.js, there is license sentences as following. I think these kind of sentences should be included in ```LICENSE``` file.
```
This license applies to parts of '*.js' files in '/src/js', implementing node.js 
   compatible API, originating from the https://github.com/node/node repository:
```
As you **imported** multiple files from IoT.js(most of the files in ```tools/common_py``` directory), it is not sufficient to specify license at the each file.

**Libraries** that we use are different from imported files. We decoupled their source code from our project using ```git submodule```. For those libraries, specifying the license in [this wiki page](https://github.com/SKKU-ESLAB/ANT/wiki/License) will be enough.

File: tools/install-deps-rpi.sh (hunk 1 above)
Comment (RedCarrottt) [reply tools/install-deps-rpi.sh]: Okay, then let's remove the file with ```target/tegraTX1/install-deps-tegraTX1.sh```.

**Comment by sinban04** [comment]
@RedCarrottt 
Okay, then you mean we need to write some comment about that in License File?
I will update my PR with the License file.

File: tools/install-deps-rpi.sh (hunk 2 above)
Comment (LeeHayun) [reply tools/install-deps-rpi.sh]: Okay. I will do.

**Pull request merged**